src/sqlancer/postgres/gen/PostgresTableGenerator.java

```diff
@@ -54,6 +55,8 @@ public PostgresTableGenerator(String tableName, PostgresSchema newSchema, boolea
         errors.add("no collation was derived for partition key column");
         errors.add("inherits from generated column but specifies identity");
         errors.add("inherits from generated column but specifies default");
+        /* this error is specific to Citus extension's columnar implementation */
+        errors.add("unlogged columnar tables are not supported");
```

Comment:
Perhaps it would make more sense to put this into the `CitusTableGenerator`? See https://github.com/sqlancer/sqlancer/blob/62a76d999f2834925a14b2921c17d34ccdc2ff47/src/sqlancer/citus/gen/CitusTableGenerator.java#L12.

Reply:
I needed to copy `createTables` and `generate` functions of Postgres to achieve this.